website/js/repl-worker.js

```diff
@@ -1,31 +1,37 @@
 self.importScripts('prepack.min.js');
 
+function onlyWarnings(buffer) {
+  return buffer.every(function(error) {
+    return error.severity === 'Warning';
```

Comment:
There are a few different severity values:
```js
export type Severity = "FatalError" | "RecoverableError" | "Warning" | "Information";
```

`"Information"` is at least as harmless is warning, so that should also be allowed.

Reply:
Great, didn't know about it, thanks.